crates/kernel/src/builtins/bf_server.rs

```diff
@@ -81,7 +80,7 @@ fn bf_notify(bf_args: &mut BfCallState<'_>) -> Result<BfRet, BfErr> {
                 event,
             },
         ))
-        .expect("scheduler is not listening");
+        .ok();
```

Comment:
Hang on, this just... silently ignores any errors? That seems bad?

Reply:
Maybe? 

Actually if we can't contact the scheduler, that's probably a panic-worth event.

I think I'll panic here.